tarogarden/interface.go

```diff
@@ -159,74 +158,6 @@ type MintingStore interface {
 		txIndex uint32, mintingProofs proof.AssetBlobs) error
 }
 
-// MintingStoreDriver represents a concrete driver of the main MintingStore
```

Comment:
Unclear why removed...waiting till thru latest set of commits to draw conclusions....

Reply:
Ah ok, I guess that we'll just use build tags w/ diff dirs for the tests kinda like we do for the `lnwallet` interface tests?

Reply:
Yeah, we just have a general `NewTestDB` that has a different implementation for the different build tags. Added a comment to this commit to clarify.